core/idrepo/logic/src/main/java/org/apache/syncope/core/logic/NotificationLogic.java

```diff
@@ -70,7 +70,8 @@ public NotificationTO read(final String key) {
 
     @PreAuthorize("hasRole('" + IdRepoEntitlement.NOTIFICATION_CREATE + "')")
     public NotificationTO create(final NotificationTO notificationTO) {
-        return binder.getNotificationTO(notificationDAO.save(binder.create(notificationTO)));
+        Notification notification = notificationDAO.save(binder.create(notificationTO));
```

Comment:
What's the rationale for this change?

Reply:
Chaining is good, but I'd say it's easier to read and understand this way.

Reply:
Hum, I don't agree completely: anyway, I still don't see the rationale of such change in the context of the current PR.

If you really want to propose such change, you should be doing it across a whole lot of Logic classes' methods, which all look as this one.

Reply:
You're right. I like things to be consistent all the way. So let me revert this.